radio/src/gui/128x64/radio_hardware.cpp

```diff
@@ -205,6 +206,24 @@ void menuRadioHardware(event_t event)
         break;
       }
 
+      case ITEM_RADIO_HARDWARE_SERIAL_BAUDRATE:
+        lcdDrawTextAlignedLeft(y, "Max bauds");
+        lcdDrawNumber(HW_SETTINGS_COLUMN2, y, CROSSFIRE_BAUDRATES[g_eeGeneral.mavbaud], attr|LEFT);
+        if (attr) {
+          g_eeGeneral.mavbaud = DIM(CROSSFIRE_BAUDRATES) - 1 - checkIncDecModel(event, DIM(CROSSFIRE_BAUDRATES) - 1 - g_eeGeneral.mavbaud, 0, DIM(CROSSFIRE_BAUDRATES) - 1);
```

Comment:
It is true that `g_eeGeneral.mavbaud` is only used on AVR radios when MAVLINK compilation option is enabled. But using it here for Crossfire I think is not appropriate. At least it needs a generous comment in datastructs. 

I would much more prefer a more clean solution, perhaps an ifdef and a different name for this variable for ARM platforms.

Reply:
I propose to rename mavbaud to telemetryBaudrate. It might be more accurate.

Reply:
Use
```c
#if defined(CPUARM)
#else
#endif
```

See how `SPLASH_MODE` is defined in `datastructs.h`.

Reply:
I used IF_AVR / IF_ARM macros instead. Hope this is fine.